README.md

```diff
@@ -57,7 +57,7 @@ in `kube-system` namespace.
 First we create a simple Docker image utilizing the Dockerfile found in the root directory:
 
 ```
-$ make image
+$ make docker
```

Comment:
Could you update README to include both `make image` and `make docker` targets and explain the difference? Also I was wondering if instead of using `docker` if we could use some other name similar to image or container-image or along these lines?

Reply:
I initially thought of renaming the `image` target to `dev-image` but didn't want to make a breaking change. If that's something you're happy with, we could have `dev-image` which builds the way it is currently before this patch, and then `image` which does the multi-stage build.

👍 on updating the README to document both targets.

Reply:
sgtm